setup.py

```diff
@@ -13,6 +14,43 @@
 
 install_requires = open("requirements.txt", encoding="utf-8").read().strip().split("\n")
 dev_requires = open("dev-requirements.txt", encoding="utf-8").read().strip().split("\n")
+dangerous_requires = (
```

Comment:
We technically don't need this file since these are the only "extras" we have. We used the file before for docker caching, but in this case we're already pinning them in the Dockerfile. It's redundant but not harmful

Reply:
ah that makes sense, it did seem like overkill. if you wanna adjust, i'm totally good with that, i'm gonna leave it as is for now though just because i've got limited bandwidth!